### PR TITLE
DOCSUP-8221: check_query_single_value_result setting

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #8338

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -2844,4 +2844,15 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
 
 Default value: `60`.
 
+## check_query_single_value_result {#check_query_single_value_result}
+
+For `MergeTree` family engines defines the level of detail for the [CHECK TABLE](../../sql-reference/statements/check-table.md#checking-mergetree-tables) query result.
```
Comment (adevyatova) [inline docs/en/operations/settings/settings.md:2849]:
```suggestion
Defines the level of detail for the [CHECK TABLE](../../sql-reference/statements/check-table.md#checking-mergetree-tables) query result for `MergeTree` family engines .
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -2844,4 +2844,15 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+
+Possible values:
+
+-   0 — the `CHECK TABLE` query shows a check status for every individual data part of a table.
```
Comment (adevyatova) [inline docs/en/operations/settings/settings.md:2853]:
```suggestion
-   0 — the query shows a check status for every individual data part of a table.
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -2844,4 +2844,15 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+-   1 — the `CHECK TABLE` query shows the general table check status.
```
Comment (adevyatova) [inline docs/en/operations/settings/settings.md:2854]:
```suggestion
-   1 — the query shows the general table check status.
```

File: docs/en/sql-reference/statements/check-table.md [hunk 4]
```diff
@@ -30,9 +30,36 @@ Performed over the tables with another table engines causes an exception.
 
 Engines from the `*Log` family don’t provide automatic data recovery on failure. Use the `CHECK TABLE` query to track data loss in a timely manner.
 
-For `MergeTree` family engines, the `CHECK TABLE` query shows a check status for every individual data part of a table on the local server.
+## Checking the MergeTree Family Tables {#checking-mergetree-tables}
 
-**If the data is corrupted**
+For `MergeTree` family engines, if [check_query_single_value_result](../../operations/settings/settings.md#check_query_single_value_result) = 0, the `CHECK TABLE` query shows a check status for every individual data part of a table on the local server.
```
Comment (adevyatova) [inline docs/en/sql-reference/statements/check-table.md:35]:
```suggestion
For `MergeTree` family engines, if the setting [check_query_single_value_result](../../operations/settings/settings.md#check_query_single_value_result) = 0, the `CHECK TABLE` query shows a check status for every individual data part of a table on the local server. 
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -2746,4 +2746,15 @@ SELECT * FROM test2;
 
 Значение по умолчанию: `60`.
 
+## check_query_single_value_result {#check_query_single_value_result}
+
+Для таблиц семейства `MergeTree` определяет уровень детализации результата для запросов [CHECK TABLE](../../sql-reference/statements/check-table.md#checking-mergetree-tables).
```
Comment (adevyatova) [inline docs/ru/operations/settings/settings.md:2751]:
```suggestion
Определяет уровень детализации результата для запросов [CHECK TABLE](../../sql-reference/statements/check-table.md#checking-mergetree-tables) для таблиц семейства `MergeTree`.
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -2746,4 +2746,15 @@ SELECT * FROM test2;
...
+
+Возможные значения:
+
+-   0 — запрос `CHECK TABLE` возвращает статус каждого куска данных таблицы.
+-   1 — запрос `CHECK TABLE` возвращает статус таблицы в целом.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2756]:
```suggestion
-   0 — запрос возвращает статус каждого куска данных таблицы.
-   1 — запрос возвращает статус таблицы в целом.
```

**Review by kssenii — APPROVED** [review]
LGTM.
`Yandex synchronization check`  failure - not related.

**Pull request merged**